benchmarks/src/main/java/io/druid/benchmark/FilterPartitionBenchmark.java

```diff
@@ -228,17 +226,15 @@ public void tearDown() throws IOException
 
   private IncrementalIndex makeIncIndex()
   {
-    return new OnheapIncrementalIndex.Builder()
+    return new IncrementalIndex.Builder()
         .setIncrementalIndexSchema(
             new IncrementalIndexSchema.Builder()
```

Comment:
Sure but it seems that such usage (i.e., setting only `metrics`) only occurs in non-productions use (i.e., tests and benchmarks). Should we add in another API only for these uses?